tools/clang/lib/SPIRV/SpirvEmitter.h

```diff
@@ -613,6 +613,12 @@ class SpirvEmitter : public ASTConsumer {
   /// Process ray query intrinsics
   SpirvInstruction *processRayQueryIntrinsics(const CXXMemberCallExpr *expr,
                                               hlsl::IntrinsicOp opcode);
+
+  /// Create SpirvIntrinsicInstruction for arbitrary SPIR-V instructions
+  /// specified by [[vk::ext_instruction(..)]] or [[vk::ext_type_def(..)]]
+  SpirvInstruction *createSpirvIntrInstExt(
+      llvm::ArrayRef<const Attr *> attrs, QualType retType, SourceLocation loc,
```

Comment:
Is it  better to put the SourceLocation to be the last function parameter?

Reply:
Done